google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -141,10 +141,9 @@ public void close(StreamWriter streamWriter) {
       }
     }
 
-    long getInflightWaitSeconds() {
+    long getInflightWaitSeconds(StreamWriter streamWriter) {
```

Comment:
why change the interface here? Could we just pass in this?

Reply:
This interface is insible to the customer, it's just that the streamwriter reference is needed by the pool to find the correcsponding worker